### PR TITLE
Exported readContext method

### LINKED ISSUE #91: Export readContext method
I came up with scenario where I need to call `readContext` from the bot code.

It is in `controller.on('event', (bot, message) => {})`
Which is used to process events sent by the client.

Since they have no text, events aren't passed through middleware, so they have no watsonData property. If I want to updateContext after processing a particular event, I have to read it first.

```js
controller.on('event', (bot, message) => {
  //readContext
  //processing
  //updateContext
  //sendToWatson
}
```

### PR DESCRIPTION
**Pull request by Naktibalda**
Closes #91

### CODE REVIEW

**Comment by Naktibalda** [comment]
If #95 is accepted, this change is no longer needed.

**Comment by Naktibalda** [comment]
Actually readContext is still necessary because some information from context can be necessary for event processing.

File: README.md [hunk 1]
```diff
@@ -221,6 +223,43 @@ controller.on('message_received', processWatsonResponse);
...
+controller.on('facebook_postback', function(bot, message) {
+   bot.reply(message, 'Great Choice!!!! (' + message.payload + ')');
+
+});
+```
+Since they usually have no text, events aren't processed by middleware and have no watsonData attribute.
+If event handler wants to make use of some data from context, it has to read it first.
+Example:
+```js
+controller.on('facebook_postback', function(bot, message) {
```
Comment (germanattanasio) [inline README.md]: The example uses 4 spaces for indentation but the code uses 2

File: README.md [hunk 2]
```diff
@@ -221,6 +223,43 @@ controller.on('message_received', processWatsonResponse);
...
+    watsonMiddleware.readContext(message.user, function(err, context) {
+        if (!context) {
+            context = {};
+        }
+        //do something useful here
+        myFunction(context.field1, context.field2, function(err, result) {
+            const newMessage = clone(message);
```
Comment (germanattanasio) [inline README.md]: I think `clone` should be required. Users may not know that it's the `clone` npm library

**Pull request merged**